### ISSUE TITLE
[bench request] OpenJeff pilot v1 — open Gemma 4 12B LoRA, evaluator-controlled runtime

### ISSUE THREAD

**Issue by greyaperez** [fstandhartinger/jevbench#54]
Please consider OpenJeff pilot v1 for an evaluator-controlled JevBench run, including the current v1.4 sealed evaluation when feasible.

OpenJeff is an experimental finite-choice decision model: Gemma 4 12B IT plus an original LoRA adapter, candidate-token probability readout, and a frozen synthetic-data temperature. The submission is AR-only; the separate diffusion experiment is not the submitted model.

## Frozen artifacts

- Code: https://github.com/greyaperez/openjeff/tree/c26040b17e6a41aa5d8d46bb6e0474c98143f25e
- Model: https://huggingface.co/greyecho/openjeff-pilot-v1/tree/1825631e9e9e1ff74bf7b9df2e099bff6bd86732
- Launch instructions and mapping disclosure: https://github.com/greyaperez/openjeff/blob/c26040b17e6a41aa5d8d46bb6e0474c98143f25e/docs/jevbench-submission.md
- Model card: https://github.com/greyaperez/openjeff/blob/c26040b17e6a41aa5d8d46bb6e0474c98143f25e/model_cards/openjeff-pilot.md

Original code, adapter, and adaptation data are Apache-2.0; upstream artifacts retain their terms. The Hub weights match the frozen GitHub adapter SHA-256.

## Running it

The documented target is NVIDIA A100 SXM 80 GB, Python 3.12, PyTorch 2.8.0+cu128, Transformers 5.17.0, PEFT 0.21.0, BF16 foundation with SDPA. Please use the documented container and pinned dependencies. The calibration identity rejects other runtimes rather than silently changing the probability claim. Minimum VRAM has not been established.

`python -m openjeff.benchmark_serve --run runs/pilot-v1 --port 8766`

This starts a loopback-only `/v1/systemone` endpoint for the upstream `typesafe` adapter: endpoint `http://127.0.0.1:8766`, model `openjeff-pilot-v1`, empty key-env. Run the harness on the evaluator machine or through an evaluator-controlled tunnel. The service does not save requests or predictions. It supports noul, choice, and score questions, full exact-label distributions, 2–26 options, and an 8,192-token diagnostic limit with no truncation. GPU rental cost should be measured by the evaluator; no zero-cost claim is made.

## Evidence and limits

The original direct evaluator scored 208/231 public items (90.04%), including 89/111 hard items (80.18%). These are historical public-subset results, not an official rank. Raw evidence and reproduction are in the repository. Training used 8,000 original rule-grounded synthetic examples, no JevBench examples or teacher API outputs. Synthetic normalized-state overlap is documented; foundation pretraining contamination is unknown.

The new HTTP endpoint preserves per-label meanings but follows received choice-criteria insertion order. The wire format omits the separate task labels order used by our original evaluator, so fresh endpoint measurements are required. Its synthetic temperature is transferred unchanged to longer inputs, not refitted on benchmark data. The 44-test suite passes (one optional model-runtime test skipped), including mapping checks across all 231 public items and mock-scorer HTTP round trips; this new endpoint has not yet been GPU-benchmarked. Prior GPU reload/parity evidence applies to the original `/v1/decide` service.

Evaluator-controlled execution is preferred so sealed inputs stay with you. Please let us know whether this package fits your evaluation workflow or needs a different integration. Thank you!

**Comment by fstandhartinger** [comment]
Thanks — Harold has logged this for review. Contributor text and code are treated as untrusted data; no external code was run on the benchmark host.

- Harold

**Comment by greyaperez** [comment]
Very much appreciated. If you have any follow up questions or anything else, let me know.
